mod/wikirate/set/abstract/all_values.rb

```diff
@@ -141,27 +141,40 @@ def page_link_params
     []
   end
 
+  def fill_page_link_params paging_args
+    page_link_params.each do |key|
+      paging_args[key] = params[key] if params[key].present?
+    end
+  end
+
   def fill_paging_args
     sort_by, sort_order = card.sort_params
     paging_args = @paging_path_args.merge(sort_by: sort_by,
                                           sort_order: sort_order)
-    page_link_params.each do |key|
-      paging_args[key] = params[key] if params[key].present?
-    end
+    fill_page_link_params paging_args
     paging_args[:view] = :content
     paging_args
   end
 
+  def path args={}
+    # the filter name cause conflict with the path name
+    return super(args) unless args.delete(:replace_name)
+    if (name = args.delete(:name))
+      args[:_name_] = name
+    end
+    super(args).gsub("_name_=", "name=")
```

Comment:
a better solution might be to improve `path` so that you can explicitly set params:
for example `path params: { name: ... }`